### PR TITLE
cmd/bosun: Export time.ParseDuration as ParseDuration to templates

### LINKED ISSUE #1014: Expose 'func (Time) Add' function in template engine
Hi,
In our use-case, we're generating Kibana URLs that include timestamps for dashboard time-window display.
Currently, I can put either fixed ```<time>-ago``` notation (4h-ago) or absolute time (from t1 to t2).

A relevant time-frame can be injected to the generated URL by taking the alert timestamp and subtracting  a duration using ```time.Add(<negative_duration>)```. the resulting Dashboard URL will always display a time-window like 1 hour before the alert until 'now'.

Thanks,
Yarden

### PR DESCRIPTION
**Pull request by kylebrandt**
Fixes #1014

### CODE REVIEW

File: cmd/bosun/conf/conf.go [hunk 1]
```diff
@@ -675,6 +675,7 @@ var defaultFuncs = ttemplate.FuncMap{
 	"short": func(v string) string {
 		return strings.SplitN(v, ".", 2)[0]
 	},
+	"ParseDuration": time.ParseDuration,
```
Comment (captncraig) [inline cmd/bosun/conf/conf.go]: lower case looks to be the convention. `parseDuration`

**Comment by captncraig** [comment]
@kylebrandt what problem does this solve. It looks like @ayashjorden solved #1014 without it.

**Comment by kylebrandt** [comment]
@captncraig See the associated issue. It makes it so we don't have to do some string replacement or ugly constants in the templates.

**Comment by captncraig** [comment]
ok, change to parseDuration and merge.

**Pull request merged**